### PR TITLE
Commits to update command-validator files.

### PR DESCRIPTION
**Pull request by davidogbudu**
These commits were made using the git bash terminal, then pushed to Github for contribution to command-validator repository.
Updates include:
- Tested and updated spec files for FSM
- Updates to manpages

### CODE REVIEW

File: spec/builtins/cd.spec.js [hunk 1]
```diff
@@ -1,73 +1,157 @@
 import { CommandValidator } from "../../src/cmd-validator.js";
 
-describe("cd command validation", () => {
+describe("ls command validation", () => {
```
Comment (justiceo) [inline spec/builtins/cd.spec.js]: "ls" command should not be in cd.spec.js

File: spec/builtins/cd.spec.js [hunk 2]
```diff
@@ -1,73 +1,157 @@
...
+
+  test("ls with multiple separated options", () => {
+    expect(validator.validateCommand("ls -a -l")).toBe(true);
+  });
+
+  test("ls with option and hidden files", () => {
+    expect(validator.validateCommand("ls -a .hidden_file")).toBe(true);
+  });
+
+  test("ls with complex combination", () => {
```
Comment (justiceo) [inline spec/builtins/cd.spec.js]: Another test with this description exists and is marked invalid.

Some of these cases look like duplicates, review and prune

File: spec/builtins/ls.spec.js [hunk 3]
```diff
@@ -1,140 +1,93 @@
 import { CommandValidator } from "../../src/cmd-validator.js";
 
-describe("ls command validation", () => {
+describe("awk command validation", () => {
```
Comment (justiceo) [inline spec/builtins/ls.spec.js]: awk should not be inside "ls"

File: spec/builtins/mv.spec.js [hunk 4]
```diff
@@ -1,65 +1,105 @@
 import { CommandValidator } from "../../src/cmd-validator.js";
 
-describe("mv command validation", () => {
+describe("rm command validation", () => {
```
Comment (justiceo) [inline spec/builtins/mv.spec.js]: rm should not be here. 

Go through the rest of the files, ensure the right command is in the right spec file, remove duplicate test cases.

**Review by justiceo — CHANGES_REQUESTED** [review]
Go through the rest of the files, ensure the right command is in the right spec file, remove duplicate test cases.

**Comment by davidogbudu** [comment]
Right away, boss

**Pull request merged**